server/processor.go

```diff
@@ -0,0 +1,164 @@
+package main
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/mattermost/mattermost-plugin-content-moderator/server/moderation"
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/mattermost/mattermost/server/public/plugin"
+	"github.com/pkg/errors"
+)
+
+const postsPerMinuteLimit = 500
```

Comment:
How was this calculated? Should we make it configurable with some recommended values?

Reply:
That's a great question. [The rate set by Azure is 1000 per minute](https://learn.microsoft.com/en-us/azure/ai-services/content-safety/faq):
![image](https://github.com/user-attachments/assets/11be37b2-9530-4bc3-b87b-f0d6d80f9d69)

I just cut that in half. I'll add a comment.

Reply:
Can we add the doc URL to this const?